processor/memorylimiterprocessor/README.md

```diff
@@ -8,58 +8,60 @@
 
 The memory limiter processor is used to prevent out of memory situations on
 the collector. Given that the amount and type of data the collector processes is
-environment specific and resource utilization of the collector is also dependent
+environment-specific and resource utilization of the collector is also dependent
 on the configured processors, it is important to put checks in place regarding
 memory usage.
  
-The memory_limiter processor allows to perform periodic checks of memory
-usage if it exceeds defined limits will begin refusing data and forcing GC to reduce
-memory consumption.
+The memory limiter processor performs periodic checks of memory
+usage and will begin refusing data and forcing GC to reduce
+memory consumption when defined limits have been exceeded.
 
-The memory_limiter uses soft and hard memory limits. Hard limit is always above or equal
-the soft limit.
+The processor uses soft and hard memory limits. The hard limit is always above or equal
+to the soft limit.
 
-When the memory usage exceeds the soft limit the processor will enter the memory limited
-mode and will start refusing the data by returning errors to the preceding component
+The processor will enter memory limited mode and will start refusing the data when
+memory usage exceeds the soft limit. This is done by returning errors to the preceding component
 in the pipeline that made the ConsumeLogs/Trace/Metrics function call.
-The preceding component should be normally a receiver.
+The preceding component should normally be a receiver.
```

Comment:
It might be easiest to say "The best practice is to add the memory limiter processor as the first processor in a pipeline."

Reply:
I found that this is stated twice in the README, so I've moved what you've suggested here to be where the second mention of this best practice was.